### ISSUE TITLE
Exception when creating posts with multiple app configs 

### ISSUE THREAD

**Issue by mammuth** [nephila/djangocms-blog#462]
I've stumbled upon an issue when creating posts.

When creating a post via admin or the corresponding menu link, I get an error after choosing the wished config (from two created ones) and submitting the form:
```
'NoneType' object has no attribute 'namespace'
Request: /de/admin/djangocms_blog/post/add/?cms_path=%2Fde%2Fnews%2F&language=de
djangocms_blog/admin.py in get_fieldsets, line 338
```

When debugging the code, I stumbled upon those findings:
https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/admin.py#L310
```
        app_config_default = self._app_config_select(request, obj)   # None
        if app_config_default is None and request.method == 'GET': # False, since we're posting
            return super(PostAdmin, self).get_fieldsets(request, obj)
        if not obj:
            config = app_config_default  # This will be taken as config, however app_config_default is None
        else:
            config = obj.app_config
```

So why `_app_config_select(request, obj)` returns None?
https://github.com/aldryn/aldryn-apphooks-config/blob/master/aldryn_apphooks_config/admin.py#L59
```
if not obj and not request.GET.get(self.app_config_attribute, False):  # True (We're posting and obj is None)
            config_model = get_apphook_model(self.model, self.app_config_attribute)
            if config_model.objects.count() == 1:   # False, since I've got 2 configs
                return config_model.objects.first()
            return None  # None will be returned
        elif obj and getattr(obj, self.app_config_attribute, False):
            return getattr(obj, self.app_config_attribute)
        elif request.GET.get(self.app_config_attribute, False):
            config_model = get_apphook_model(self.model, self.app_config_attribute)
            return config_model.objects.get(
                pk=int(request.GET.get(self.app_config_attribute, False))
            )
        return False
```

So I'm wondering how the selection/setting of the blog config should work here? I'm probably not understanding something crucial?

How can I further debug the underlying issue?

Environment information
- 2 blog configs
- Divio Cloud project
- Single language
- django 1.8.19
- django-cms 3.5.2
- djangocms-blog 0.9.7
- aldryn-aphook-config 0.4.1

**Comment by yakky** [comment]
@mammuth it's a bug in `aldryn-apphooks-config` preveting the apphooks config selection to work properly
I should be able to push a fix there soon

**Comment by mammuth** [comment]
Thanks for digging into this @yakky! I'm looking forward to the fix.

**Comment by yakky** [comment]
I opened the PR, hopefully we will be able to be merged soon

**Comment by yakky** [comment]
Fixed by aldryn-apphooks-config==0.5.2, closing this
